larray/inout/xw_excel.py

```diff
@@ -601,6 +637,8 @@ def load(self, header=True, convert_float=True, nb_axes=None, index_col=None, fi
 
     # XXX: deprecate this function?
     def open_excel(filepath=None, overwrite_file=False, visible=None, silent=None, app=None, load_addins=None):
+        # unsure if _create_instance is reliable enough to be the default behavior or not
+        _create_instance('Excel.Application')
```

Comment:
should only be called when creating a new app/excel instance!!!

Reply:
You said "this special create_instance code should be used in/moved to open_excel" in your last comment.